public/emails/new_user_invite.txt

```diff
@@ -0,0 +1,9 @@
+{{Subject .Subject "{{.InvitedBy}} has invited you to join Grafana"}}
+
+You're invited to join {{.OrgName}}
+
+You've been invited to join the {{.OrgName}} organization by {{.InvitedBy}}. To accept
+your invitation and join the team, please copy/paste the link below into your browser
```

Comment:
```suggestion
your invitation and join the team, copy and paste the link below into your browser
```

Reply:
@oddlittlebird Note that I have just copied the contents from the HTML templates so if you want this to be changed here, it should be changed in the HTML as well.

Reply:
Yes, please do.

Reply:
Done